scheduler/src/cook/mesos/util.clj

```diff
@@ -338,20 +346,67 @@
 (def ^:const job-states #{"running" "waiting" "completed"})
 (def ^:const instance-states #{"success" "failed"})
 
+;; get-jobs-by-user-and-state-and-submit-by-state is a bit opaque
```

Comment:
This function name is a bit of a mouthful. Maybe just `get-jobs-by-user-and-state`?

Reply:
Renamed shorter